pyvo/dal/tests/test_tap.py

```diff
@@ -190,6 +201,67 @@ def result(self, request, context):
         self.validator(request)
         return get_pkg_data_contents('data/tap/obscore-image.xml')
 
+    def get_job(self, request, context):
+        self.validator(request)
+        jobid = int(job_re_path.match(request.path).group(1))
+
+        job = JobFile()
+        job.jobid = jobid
+        job.phase = 'EXECUTING'
+        job.ownerid = '222'
+        job.creationtime = Time.now()
+        io = BytesIO()
+        job.to_xml(io)
+        return io.getvalue()
+
+    def _get_jobref_rep(self, jobid, phase, runid, ownerid, creation_time):
+        doc = '    <uws:jobref id="{}">\n'
+        doc += '        <uws:phase>{}</uws:phase>\n'
+        doc += '        <uws:runId>{}</uws:runId>\n'
+        doc += '        <uws:ownerId>{}</uws:ownerId>\n'
+        doc += '        <uws:creationTime>{}</uws:creationTime>\n'
+        doc += '    </uws:jobref>\n'
```

Comment:
Maybe collapse this down to a multi line string rather than doing this appending?

Reply:
Done